packages/migrations/src/actions/down/2023.01.12T17.00.23.cdn-tokens.sql

```diff
@@ -0,0 +1 @@
+raise 'down migration not implemented'
```

Comment:
I think it worth implementing it?

Reply:
Down migrations are never worth implementing imho